doc/api/fs.md

```diff
@@ -2697,10 +2697,10 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronously rename file at `oldPath` to the pathname provided
-as `newPath`. In the case that `newPath` already exists, it will
-be overwritten. No arguments other than a possible exception are
-given to the completion callback.
+Asynchronously rename file or directory at `oldPath` to the pathname provided
+as `newPath`. In the case that `newPath` is a file and already exists, it will
+be overwritten. If there is a directory at `newPath` an error will be raised instead.
```

Comment:
Linter nit: this line needs to be wrapped to not exceed 80 characters.

Reply:
As described in #https://github.com/nodejs/node/issues/21957#issuecomment-408486653 that is actually not always true. Unix systems will not raise an exception if the directory is empty. That is only the case for Windows. I did not check but that should hold true for all OS.